cookbooks/bcpc/recipes/graphite.rb

```diff
@@ -47,48 +47,48 @@
   python-scandir
   libcairo2-dev
   libffi-dev
-}.each do |pkg|
+).each do |pkg|
   package pkg do
     action :upgrade
   end
 end
 
 package 'python-django' do
   action :upgrade
-  version node[:bcpc][:graphite][:django][:version] 
+  version node['bcpc']['graphite']['django']['version']
 end
 
-%w{
+%w(
   python-whisper
   python-carbon
   python-graphite-web
-}.each do |pkg|
+).each do |pkg|
   package pkg do
     action :upgrade
   end
 end
 
-%w{
+%w(
   cache
   relay
   aggregator
-}.each do |pkg|
+).each do |pkg|
   template "/etc/init.d/carbon-#{pkg}" do
     source 'carbon/init.erb'
     owner 'root'
     group 'root'
-    mode 00755
+    mode 0o755
```

Comment:
Just mentioning this once to avoid many redundant comments, but when changing `0755` to an octal literal, the resulting number should be `0o0755` to preserve the first 0 bit - removing any setuid, sticky, etc flags.